packages/@okta/vuepress-theme-prose/components/Quickstart.vue

```diff
@@ -1,265 +1,87 @@
+          '#/react/dotnet/aspnet4' : '/docs/guides/sign-into-spa/react/before-you-begin/',
+          '#/vue/nodejs/express' : '/docs/guides/sign-into-spa/vue/before-you-begin/',
+          '#/vue/nodejs/generic' : '/docs/guides/sign-into-spa/vue/before-you-begin/',
+          '#/vue/java/spring' : '/docs/guides/sign-into-spa/vue/before-you-begin/',
+          '#/vue/java/generic' : '/docs/guides/sign-into-spa/vue/before-you-begin/',
+          '#/vue/php/generic' : '/docs/guides/sign-into-spa/vue/before-you-begin/',
+          '#/vue/dotnet/aspnetcore' : '/docs/guides/sign-into-spa/vue/before-you-begin/',
+          '#/vue/dotnet/aspnet4' : '/docs/guides/sign-into-spa/vue/before-you-begin/',
+          '#/android/nodejs/express' : '/docs/guides/sign-into-mobile-app/android/before-you-begin/',
+          '#/android/nodejs/generic' : '/docs/guides/sign-into-mobile-app/android/before-you-begin/',
+          '#/android/java/spring' : '/docs/guides/sign-into-mobile-app/android/before-you-begin/',
+          '#/android/java/generic' : '/docs/guides/sign-into-mobile-app/android/before-you-begin/',
+          '#/android/php/generic' : '/docs/guides/sign-into-mobile-app/android/before-you-begin/',
+          '#/android/dotnet/aspnetcore' : '/docs/guides/sign-into-mobile-app/android/before-you-begin/',
+          '#/android/dotnet/aspnet4' : '/docs/guides/sign-into-mobile-app/android/before-you-begin/',
+          '#/ios/nodejs/express' : '/docs/guides/sign-into-mobile-app/ios/before-you-begin/',
+          '#/ios/nodejs/generic' : '/docs/guides/sign-into-mobile-app/ios/before-you-begin/',
+          '#/ios/java/spring' : '/docs/guides/sign-into-mobile-app/ios/before-you-begin/',
+          '#/ios/java/generic' : '/docs/guides/sign-into-mobile-app/ios/before-you-begin/',
+          '#/ios/php/generic' : '/docs/guides/sign-into-mobile-app/ios/before-you-begin/',
+          '#/ios/dotnet/aspnetcore' : '/docs/guides/sign-into-mobile-app/ios/before-you-begin/',
+          '#/ios/dotnet/aspnet4' : '/docs/guides/sign-into-mobile-app/ios/before-you-begin/',
+          '#/react-native/nodejs/express' : '/docs/guides/sign-into-mobile-app/react-native/before-you-begin/',
+          '#/react-native/nodejs/generic' : '/docs/guides/sign-into-mobile-app/react-native/before-you-begin/',
+          '#/react-native/java/spring' : '/docs/guides/sign-into-mobile-app/react-native/before-you-begin/',
+          '#/react-native/java/generic' : '/docs/guides/sign-into-mobile-app/react-native/before-you-begin/',
+          '#/react-native/php/generic' : '/docs/guides/sign-into-mobile-app/react-native/before-you-begin/',
+          '#/react-native/dotnet/aspnetcore' : '/docs/guides/sign-into-mobile-app/react-native/before-you-begin/',
+          '#/react-native/dotnet/aspnet4' : '/docs/guides/sign-into-mobile-app/react-native/before-you-begin/',
+        }
       }
     },
 
     beforeMount: function () {
       if(window.location.hash == "") {
```

Comment:
I don't see anything alarming, but just double-checking: have you tested this with a build and static server to ensure server-side rendering doesn't break anything here?